cdap-ui/app/directives/breadcrumbs/breadcrumbs.less

```diff
@@ -0,0 +1,197 @@
+@import "../../styles/variables.less";
+@import "../../styles/themes/cdap/mixins.less";
+@import "../../../bower_components/bootstrap/less/mixins.less";
+
+my-breadcrumbs {
+  @media (max-width: @screen-sm-max) {
+    display: none;
+  }
+  @media (min-width: @screen-md-min) {
+    background-color: white;
+    border-bottom: 1px solid @panel-default-border;
+    position: absolute;
+    top: 75px;
+    right: 0;
+    left: 0;
+    height: 40px;
+    line-height: 40px;
+    .crumbs {
+      .flexbox();
+      flex-direction: row;
+      flex-wrap: nowrap;
+      align-items: center;
+      width: 100%;
+      height: 40px;
+
+      li.dropdown {
+        ul.dropdown-menu {
+          width: 120px;
+          max-height: 290px;
+          min-width: 0;
+          background-color: white;
+          border-color: white;
+          padding: 10px 0;
+          margin-top: 1px;
+          .border-radius(4px);
+          .box-shadow(0 6px 12px rgba(0, 0, 0, 0.175));
+          overflow-x: hidden;
+          overflow-y: scroll;
+          @media (max-width: @screen-md-max) {
+            position: absolute;
+          }
+          > li {
+            background-color: white;
+            padding: 0;
+            width: 120px;
+            &:focus {
+              outline: 0;
+            }
+            > a {
+              color: @cdap-header;
+              font-weight: 500;
+              background-color: white;
+              padding-right: 10px;
+              padding-left: 10px;
+              &:hover, &:focus {
+                color: @cdap-orange;
+                cursor: pointer;
+              }
+              &:focus {
+                outline: 0;
+              }
+            }
+            i {
+              display: inline-block;
+              width: 13px;
+            }
+            .active {
+              color: #c4c4c4;
+              pointer-events: none;
+            }
+          }
+        }
+        &.open {
+          a.ns-dropdown-toggle:focus {
+            border-color: transparent;
+          }
+        }
+      }
+    }
+    ul.nav {
+      margin: 0;
+      white-space: nowrap;
+      .order(1);
+      .open {
+        > a, > a:hover, > a:focus {
+          background-color: transparent;
+        }
+      }
+    }
+
+    a.ns-dropdown-toggle {
+      color: @cdap-gray;
+      font-weight: 500;
+      padding: 1px 0 0;
+      position: relative;
+      top: -1px;
+      &:hover, &:focus {
+        background-color: transparent;
+      }
+      &:after {
+        content: url('@{img-path}/chevron_front.png');
+        overflow: hidden;
+        position: absolute;
+        top: 1px;
+        height: 40px;
+        z-index: 99;
+      }
+      > span {
+        display: inline-block;
```

Comment:
Do we need to make span element inline-block? Its already an inline element right?